scripts/update-crd-reference/main.sh

```diff
@@ -2,16 +2,22 @@
 
 set -e
 
-CRD_DOCS_GENERATOR_VERSION=0.4.0
+CRD_DOCS_GENERATOR_VERSION=0.4.0-ee74b086d228a80d7c560ffe5a90bf7480c98cd7
 DESTINATION=src/content/ui-api/management-api/crd
 
 # Clear output folder
 find ${DESTINATION} -type f -not -name "_index.md" | xargs rm
 
+# Determine apiextensions version
+VERSION=$(docker run --rm -ti --entrypoint /bin/sh quay.io/giantswarm/docs-scriptrunner -c "curl --silent https://api.github.com/repos/giantswarm/apiextensions/releases/latest|jq -r .tag_name|tr -d '\n'")
+
+echo "Version: ${VERSION}"
```

Comment:
IMO this should be moved into the code of crd-docs-generator.

Reply:
Is this a blocker for you? If yes, please explain why it shouldn't be done here.

Reply:
Spinning up a container for just a single `curl | jq` call feels a little redundant to me. IMO this belongs into application code. For example the absence of the new flag you've implemented in crd-docs-generator could mean: "Use the latest version of apiextensions". The crd-docs-generator interacts with GitHub after all.

No blocker, but still, feels a little patchwork

Reply:
OK. If you could approve this and https://github.com/giantswarm/crd-docs-generator/pull/33 I would like to merge, then add the follow-up change.